src/layout/section/mediaContentSection/index.tsx

```diff
@@ -52,6 +52,10 @@ const StyledMediaContentSection = styled(MediaContentSection)`
     & .kirk-media-content-wrapper {
       align-items: center;
     }
+
+    & .kirk-media-content-title {
```

Comment:
Instead of using classes why not use **styled-components** redefinition?
```
const SubHeaderTitle = styled(TextDisplay1)`
margin: ${space.xl} 0 ${space.m};
`
<SubHeaderTitle> {title} </SubHeaderTitle>
```

Reply:
What do you think of the following?

```
${TextDisplay1} {
  margin: ${space.xl} 0 ${space.m};
}
```

Using your solution (if I undestand correcly), I'll have to create another component directly inside the `MediaContentSection` ?

Reply:
The first solution is the official styled-components sytaxe. Yes `SubHeaderTitle` would be a component inside `SubHeader`

Reply:
Updated the fix. Can you take another look?